backend/routes/user.go

```diff
@@ -209,3 +210,21 @@ func changeUsernameDevnet(w http.ResponseWriter, r *http.Request) {
 
 	routeutils.WriteResultJson(w, "Username changed")
 }
+
+func getUserColorVote(w http.ResponseWriter, r *http.Request) {
+	address := r.URL.Query().Get("address")
+	if address == "" {
+		routeutils.WriteErrorJson(w, http.StatusBadRequest, "Missing address parameter")
+		return
+	}
+
+	vote, err := core.PostgresQueryOne[int]("SELECT color_key FROM ColorVotes WHERE user_address = $1", address);
```

Comment:
We really want the color vote from the current day, so you'll need to filter on the day index like this https://github.com/keep-starknet-strange/art-peace/blob/25d773412f343fb04092d061282e819c30149b51/backend/routes/quests.go#L223